serializer.rst

```diff
@@ -23,37 +23,45 @@ install the serializer before using it:
 
 Using the Serializer Service
 ----------------------------
+The serializer's constructor takes two arguments:
 
-Once enabled, the serializer service can be injected in any service where
-you need it or it can be used in a controller::
+* an array of :ref:`normalizers <component-serializer-normalizers>`
+* an array of :ref:`encoders <component-serializer-encoders>`
+
+You can either instantiate the serializer directly:
 
     // src/Controller/DefaultController.php
     namespace App\Controller;
-
+    
     use Symfony\Bundle\FrameworkBundle\Controller\Controller;
-    use Symfony\Component\Serializer\SerializerInterface;
+    use Symfony\Component\Serializer\Serializer;
+    use Symfony\Component\Serializer\Normalizer\DateTimeNormalizer;
+    use Symfony\Component\Serializer\Encoder\CsvEncoder;
 
     class DefaultController extends Controller
     {
-        public function index(SerializerInterface $serializer)
+        public function index()
         {
-            // keep reading for usage examples
+            $serializer = new Serializer(array(new DateTimeNormalizer('Y-m-d')), array(new CsvEncoder()));
```

Comment:
👎, when using the full stack framework, the service registered by FrameworkBundle should alway be used. Here you don't benefit from the cache layer, groups, max depth...
Initializing the component manually belongs to the component docs.